Convert/src/Controller/Conversion.java

```diff
@@ -0,0 +1,116 @@
+package Controller;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import Model.*;
+
+public class Conversion {
+
+	private SubUnit basePrefixUnit;
+	private SubUnit baseMesure;
+	private ArrayList<Unit> unitsList;
+	
+	public Conversion(String fileName) throws Exception{
+		super();

```

Comment:
pas nécessaire; je crois que le compilateur l'ajoute automatiquement

Reply:
Exacte sauf si on veut utiliser un constructeur avec paramètres

Reply:
je réitère, le super n'est pas nécessaire; le compilateur, quand il y a qu'un seul constructeur, fais automatiquement l'appel au super(); vire le et tu verras que le compilateur ne lève pas d'erreur; surtout que ta classe parente c'est object